src/SuggestionsOverlay.js

```diff
@@ -82,7 +82,7 @@ class SuggestionsOverlay extends Component {
 
     return (
       <div
-        {...inline(style, { position, left, top })}
+        {...inline({ position: position || 'absolute', left, top }, style)}
```

Comment:
the switch of order here allows users to potentially override the assigned position, e.g., for rendering the suggestions below the input instead of as an overlay